tests/e2e/tests/pilot/pilot_test.go

```diff
@@ -68,8 +68,8 @@ func init() {
 		fmt.Sprintf("Auth mode for the tests (Choose from %s, %s, %s)", authModeEnable, authModeDisable, authModeBoth))
 	flag.BoolVar(&config.Mixer, "mixer", config.Mixer, "Enable / disable mixer.")
 	flag.BoolVar(&config.V1alpha1, "v1alpha1", config.V1alpha1, "Enable / disable v1alpha1 routing rules.")
-	flag.BoolVar(&config.V1alpha2, "v1alpha2", config.V1alpha2, "Enable / disable v1alpha2 routing rules.")
-	flag.BoolVar(&config.RDSv2, "rdsv2", false, "Enable RDSv2 for v1alpha2")
+	flag.BoolVar(&config.V1alpha3, "v1alpha3", config.V1alpha3, "Enable / disable v1alpha3 routing rules.")
+	flag.BoolVar(&config.RDSv3, "rdsv3", false, "Enable RDSv3 for v1alpha3")
```

Comment:
RDSv2 != v1alpha2/3. I think the v2 refers to the using the Envoy v2 API, not to v1alpha2/3. This should stay `RDSv2` in all references.

Reply:
did some test failed? (I would hope so but...)